macros/DOMxRef.ejs

```diff
@@ -5,13 +5,13 @@
  * a function.
  * @param
  *   The path of the page to link to, relative to the Web/API/ documentation path.
- * @param [optional] 
+ * @param [optional]
  *   The text to use for the link.  If omitted, the value of the first
  *   parameter will be used
- * @param [optional] 
- *   An anchor to link to on the page. Link text will display as $0.$2 or $1.$2 
- * @param [optional] 
- *   If set, do not put the domxref text in code 
+ * @param [optional]
+ *   An anchor to link to on the page. Link text will display as $0.$2 or $1.$2
+ * @param [optional]
+ *   If set, do not put the domxref text in code
```

Comment:
In general, I would prefer that you do not make minor formatting cleanups like this. What you're proposing is basically a one-line patch, and having these other changes is a distraction. Not a big deal, and no need to change it for this PR. You may have your editor configured to automatically remove trailing whitespace, but be aware that it makes reviews harder.

Reply:
This was indeed done by my editor (it trimmed the trailing whitespace)